src/components/AuthenticatedContent/AuthenticatedContent.tsx

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+
+import React from "react";
+
+const AuthenticatedContent = ({ name, email }) => {
+  console.log("Received props:", { name, email });
+  return (
+    <div>
+      <h2>Welcome to Your Dashboard, {name} </h2>
+      <p>Email: {email}</p>
+      <p>You currently do not belong to any group.</p>
+      <button onClick={() => (window.location.href = "/create-group")}>
+        Create Group
+      </button>
```

Comment:
When working inside React and using React Router Dom to create Frontend Routes, we should use the special `<Link>` component instead of modifying the `location` or using a regular `<a>` tag to move around within the site. (We can still use `<a>` as normal to link to addresses outside of our app)

This is because we want to bypass all of the initial file loading that that browsers do when loading a url for the first time in the session. Single Page Apps already have all the initial data we need from the page we're currently on, and if we need anything extra we'll fetch it manually from the API

```suggestion
      <Link to={"/create-group"}>Create Group</Link>
```

Reply:
Thanks for the explanation! In this case, though, I want a button element in the UI and not a link. Is using a `<Link>` component the only fix?

Reply:
> I want a button element in the UI and not a link

Since this is a link to another route in the app maybe the `<link>` would be more accurate in terms of functionality, but if you really want a `<button>` visually, you could always put one as a child of the `<link>` tag

Otherwise if you just want the link to look and feel like a button I think it might be more common to get that result via css: https://codepen.io/dargaCode/full/LYqvJRP